app/ui/browser-modern/sagas/ui-sagas.js

```diff
@@ -70,12 +70,12 @@ function* setURLBarValue({ urlbar, value, doc, options = { keepSelection: true }
   }
 }
 
-function* showDownloadNotification({ filename, status }) {
+function* showDownloadNotification({ path, filename, status }) {
   let message;
   const title = `Download ${status === 'success' ? 'completed' : 'failed'}`;
 
   if (status === 'success') {
-    message = `Download for ${filename} is completed.`;
+    message = `Download for ${filename} is completed. Can be located at ${path}/${filename}`;
```

Comment:
Instead of "Can be located at...", to save space, can't we just write "... at:"?
"Download for ${filename} completed at ${path}"